README.markdown

```diff
@@ -38,9 +44,9 @@ bootstrapped:
 
     git clone git://github.com/rspec/rspec-dev.git
     cd rspec-dev
-    bundle install --binstubs
-    bin/rake setup
-    bin/rake # runs tests in every repository
+    bundle install --path=vendor/bundle
```

Comment:
Why the change here and the specified `--path`?  Users can use any `--path` they want (or none).  That's a personal setting and not something that seems to warrant a recommendation here.